web/templates/package/show.html.eex

```diff
@@ -6,7 +6,12 @@ licenses = @package.meta["licenses"] || []
 links = Enum.to_list(@package.meta["links"] || [])
 build_tools = @current_release && @current_release.meta["build_tools"] || []
 
-tools = [mix: "mix.exs", rebar: "rebar.config", erlang_mk: "erlang.mk"]
+tools = if Enum.any?(build_tools, fn(tool) -> Enum.member?(["rebar", "rebar3", "make"], tool) end) do
+           [mix: "mix.exs", rebar: "rebar.config", erlang_mk: "erlang.mk"]
+        else
+           [mix: "mix.exs", erlang_mk: "erlang.mk"]
+        end
+
```

Comment:
I got an error `the Access calls for keywords expect the key to be an atom, got: "mix"`. How about using a map?

```
tools = [mix: "mix.exs", rebar: "rebar.config", erlang_mk: "erlang.mk"]
tool_compatibility = %{
  "mix" => [:mix],
  "rebar" => [:mix, :rebar, :erlang_mk],
  "rebar3"=> [:mix, :rebar, :erlang_mk],
  "erlang.mk" => [:mix, :erlang_mk]
}

compatible_tools = Enum.flat_map( build_tools, &(Map.get(tool_compatibility, &1, [])))
                   |> Enum.uniq

tools = Keyword.take(tools, compatible_tools)
```

I've updated the commit in the PR.